cross/libgpg-error/Makefile

```diff
@@ -14,3 +14,7 @@ LICENSE  = LGPL
 GNU_CONFIGURE = 1
 
 include ../../mk/spksrc.cross-cc.mk
+
+ifeq ($(findstring $(ARCH), $(ARM_ARCHES)),$(ARCH))
+CONFIGURE_ARGS = --target=arm-unknown-linux-gnueabi --host=arm-unknown-linux-gnueabi
```

Comment:
Now I noticed this came back in another PR over here: https://github.com/SynoCommunity/spksrc/pull/1811/files#r39625290.
I have to say, this looks better than copying a file. I assume you tested this for `all-supported` and `all-legacy` (as far as build errors at least)?